app/controllers/matches_controller.rb

```diff
@@ -61,11 +59,12 @@ def update
     update_connections = params[:commit]&.downcase&.include?('edit who this match connects')
     if @match.update(match_params)
       if save_and_continue || update_connections
-        redirect_to edit_match_path(@match, edit_connection_mode: update_connections ? true : false),
-                    notice: 'Match was successfully updated.'
+        redirect_to(
+          edit_match_path(@match, edit_connection_mode: update_connections ? true : false),
```

Comment:
@exbinary — I didn't think any of the rules you mentioned in the discription would add the `(` after `redirect_to` here. Do you know which one does this?

Reply:
Ah yes, good question! This is something i tweaked manually. The rule at work here is `Layout/ArgumentAlignment`, which forces any multi-line arguments to be indented one level after the method call, not aligned under the first argument as in the original code. But with that automated change, i find the code to be less scannable:
```ruby
# original form -- disallowed by Layout/ArgumentAlignment
redirect_to edit_match_path(@match, edit_connection_mode: update_connections ? true : false),
            notice: 'Match was successfully updated.'

# autocorrected form -- i find this to be imbalanced and harder to scan
redirect_to edit_match_path(@match, edit_connection_mode: update_connections ? true : false),
  notice: 'Match was successfully updated.'

# balanced form -- i prefer this form where the first argument is on a newline, but it requires parentheses
redirect_to(
  edit_match_path(@match, edit_connection_mode: update_connections ? true : false),
  notice: 'Match was successfully updated.'
)
```
Does that make sense? Curious your thoughts on this.

Reply:
In discussing this further with @exbinary, I realized my objection was that I keep misreading this as a three-argument call to `edit_match_path` that is indented weirdly. I have no objection to the guideline that's used here, and my confusion is more due to the ternary operator or the fact that `redirect_to` is multi-line when its nested `edit_match_path` isn't